InputTime/index.jsx

```diff
@@ -84,8 +86,8 @@ const InputTime = ({
       marginBottom: 0,
       marginLeft: 0,
       padding: 0,
-      '-webkit-appearance': 'none',
-      '-moz-appearance': 'none',
+      WebkitAppearance: 'none',
```

Comment:
Yeah! I'm curious what your hunch might be regarding styling `select` elements. I wonder if we could approach `InputTime` like we do with `Button` and have `InputTime` and then `<InputTime small />` -- both using the same UI except `small` being similar in size to what we have in the screenshot above. 🤔